src/library/Cardano/CoinSelection/Migration.hs

```diff
@@ -34,8 +35,13 @@
 -- that a wallet is not "fragmented enough").
 
 module Cardano.CoinSelection.Migration
-    ( depleteUTxO
+    (
+      -- * Coin Selection for Migration
+      selectCoins
+
+      -- # Internal Functions
```

Comment:
not `*` ?

Reply:
Using `#` here means that it is not processed by Haddock. (Users can still import the functions, but they're not described in the generated Haddock documentation.)